packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -214,6 +214,20 @@ class SharingCollection extends DocumentCollection {
       uri`/sharings/${sharing._id}/recipients/${recipientIndex}`
     )
   }
+
+  /**
+   * Revoke only one group of the sharing.
+   *
+   * @param {object} sharing Sharing Object
```

Comment:
is it this one ? https://github.com/cozy/cozy-client/blob/28480d5ee61e715e4f0e005813417e7b253f691d/packages/cozy-stack-client/src/SharingCollection.js#L73-L84 

If yes, maybe we can create the type and use it instead of `object`?

Reply:
No, it didn't match that one, but you're right to point out the lack of type. I've added the description of the object which was an `io.cozy.sharing` document